app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -708,6 +728,42 @@ class StatePlayerRecyclerViewAssembler private constructor(
     }
   }
 
+  private fun createBannerConfetti() {
+    val x = bannerConfettiView!!.width / 3
+    val y = bannerConfettiView!!.height / 2
+    val colorsList = confettiColors.map { getColor(bannerConfettiView.context, it) }
+    val speedMin = 2f
+    val speedMax = 4f
+    val timeToLive = 2000
+    val shapesArray = arrayOf(Square, Circle)
+    val size = Size(7)
```

Comment:
Please add a unit to the name (is this in px, dp, sp?).

Reply:
Done